### PR TITLE
Set route for users and posts

### PR DESCRIPTION
**Pull request by koachcode**
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Important: do NOT use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
- Make sure that your project has Postgres database set up.
- For each URL you should create:

-     A route.
-     An action in the correct controller.
-     A view file.

- View files should include only placeholders written in plain HTML, e.g:
  -       Here is a list of posts for a given user
  -       Make sure that you added code to handle only the 4 required URLs.
  
Make sure that when you run the rails routes command in your console, you can see the following output:
routes

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 coder,

You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight
- The PR title and description text looks good👍
- No linter errors👍
- The code is good 👍


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly add a personalized readme file and include all the necessary commands required for running the project on the user's local machine.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Instead of writing route one by one, the code below can save you.
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,78 @@
-# README
+# Ruby capston : catalog to my things
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Let's add a title that matches the project at hand, the title and the video do not represent the current project. This is a Rails project and not a Ruby capstone console app.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Ginohmk, 

Great work on making the changes I requested👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (koachcode) [reply README.md]: oh sorry about that I did not edit it properly, I will make the changes now.thanks

**Review by mmsesay — APPROVED** [review]
Hi @Ginohmk,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by koachcode** [comment]
Thank you @mmsesay @elisha2kyakpo1

**Pull request merged**